deploy/cht_sync/templates/postgrest.yaml

```diff
@@ -0,0 +1,39 @@
+# templates/postgrest.yaml
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cht-sync-postgrest
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cht-sync
+      inner.service: postgrest
+  template:
+    metadata:
+      labels:
+        app: cht-sync
+        inner.service: postgrest
+    spec:
+      automountServiceAccountToken: false
+      containers:
+        - name: postgrest
+          image: postgrest/postgrest
+          resources:
+            limits:
+              cpu: {{ (.Values.postgrest).cpu_limit | default "1000m" }}
+              memory: {{ (.Values.postgrest).memory_limit | default "500Mi" }}
+          env:
+            - name: PGRST_DB_URI
+              value: |
```

Comment:
This causes the error below. Setting the value to `{{ printf "postgres://%s:%s@%s:5432/%s" .Values.postgres.user .Values.postgres.password .Values.postgres.host .Values.postgres.db }}` works though.
```
{"code":"PGRST000","details":"connection to server at \"postgres\" (172.20.202.161), port 5432 failed: FATAL:  database \"data\n\" does not exist\n","hint":null,"message":"Database connection error. Retrying the connection."}
```